src/components/UriViewer.js

```diff
@@ -139,7 +139,7 @@ const UriViewer = ajaxCaller(class UriViewer extends Component {
           els.cell([
             link({
               target: 'blank',
-              href: bucketBrowserUrl(bucket)
+              href: bucketBrowserUrl(gsUri.match(/gs:\/\/(.+)\//)[1])
```

Comment:
Out of curiosity, what was the motivation for this change?

Reply:
This way it'll go to the correct folder in the bucket, rather than just the root.